src/shims/shims.proj

```diff
@@ -2,6 +2,35 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="dir.props" />
 
+  <PropertyGroup>
+    <PackageConfigurations>
+      netcoreapp2.0;
+    </PackageConfigurations>
+    <BuildConfigurations>
+      $(BuildConfigurations);
+      $(PackageConfigurations);
+    </BuildConfigurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Implementation shims to include in the compat pack produced by this project -->
+    <ShimsToInclude Include="System.Configuration.dll" />
```

Comment:
I bet you can construct this list from the NETFXReference item.  See below for a suggestion.

Reply:
NETFxReference has more items than the ones we need, this is a reduced list of NETFxReference, that is why I didn't construct it from there, we don't need/want all the netfx shims in the package

Reply:
Is it reduced in a way that's already stated somewhere that you can codify, or is it just an arbitrary static list that has to be recorded?

Reply:
They way that I found the list of shims we needed was by running VerifyClosure on the compat pack assemblies. So that is why I had the static list.

Reply:
Not sure I follow: compat pack assemblies don't reference the shims, its the other way around.  Are you saying you manually calculated the set of shims that had type forwards to the compat pack assemblies?  I guess that would work, but that just creates a place where things might be incorrect if someone adds a type that lives in a new place and forgets to rerun your manual process.  We try to avoid static lists backed by external processes wherever we can.

Reply:
yes I manually calculated the set of shims that had type forwards to the compat pack. 

So is t OK to have more shins than the ones needed in the package? I was trying to avoid making the package fatter and having unnused assemblies there.

Reply:
Yes it is OK.  I'd prefer it as I mentioned before.